src/web/lib/api.tsx

```diff
@@ -46,10 +48,19 @@ interface ApiResponse<T> {
 	error?: Error;
 }
 
-export const useApi = <T,>(
-	url: string,
-	options?: FetchOptions,
-): ApiResponse<T> => {
+/**
+ * @description
+ * A custom hook to make a GET request using the given url
+ * returning { loading, error, data }
+ * @param {String} url - The url to fetch
+ * @param {Object} options
+ * @param {Number} options.method - If supplied, the api will be called using this value
+ * @param {Number} options.headers - If supplied, the api will be called using this value
+ * @param {Number} options.body - If supplied, the api will be called using this value
+ * @param {Number} options.credentials - If supplied, the api will be called using this value
+ * @param {Number} options.pollInterval - If supplied, the api will be polled at this interval
+ * */
+export const useApi = <T,>(url: string, options?: Options): ApiResponse<T> => {
```

Comment:
I was unsure what the `,` was after the `T`, but found this https://stackoverflow.com/a/56989122/7378674

Reply:
I didn't put that there, prettier did and prettier knows best 🙇